source/components/header/header-component.js

```diff
@@ -1,99 +1,42 @@
-import React, { PureComponent, Fragment } from 'react';
+import React, { memo } from 'react';
 import PropTypes from 'prop-types';
-import CSSModules from 'react-css-modules';
 import cx from 'classnames';
 
-import { omit } from '../../helpers';
-
 import styles from './header.styl';
-import Button from '../button';
-import Icon from '../icon';
-import Select from '../select';
-
-class Header extends PureComponent {
-  static propTypes = {
-    user: PropTypes.string,
-    items: PropTypes.arrayOf(
-      PropTypes.shape({
-        name: PropTypes.string.isRequired,
-        value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
-        icon: PropTypes.string.isRequired,
-      })
-    ),
-    onSelect: PropTypes.func,
-    onLogout: PropTypes.func,
-    isMobile: PropTypes.bool,
-    onLogoClick: PropTypes.func,
-  };
 
-  static defaultProps = {
-    isMobile: false,
-  };
+const Header = ({ className, contentClassName, showLogo, logo, onLogoClick, children, ...elementProps }) => {
```

Comment:
Fiz algumas mudanças no componente além do que tinhámos pensado anteriormente. Mudei a ideia do "isMobile" para "showLogo" para deixar mais genérico, caso alguém queira usar a versão com logo sempre, além da versão mobile.

Além disso, coloquei o children para aparecer do lado direito do logo, para que possamos passar o que for necessário para aquela área!